vtkext/private/module/vtkF3DEXRReader.cxx

```diff
@@ -134,6 +197,24 @@ void vtkF3DEXRReader::ExecuteDataWithInformation(vtkDataObject* output, vtkInfor
         dataPtr += 3;
       }
     }
+  };
+
+  try
+  {
+    assert(this->InternalFileName);
```

Comment:
i think this assert should not be here for the memory stream part. BTW it may be better to make it an actual if test since we now can have an empty InteralFileName, I think.

Reply:
Yes when loading a USD scene the EXRReader will have an empty filename. Interestingly in the test case TestF3DEXRMemReader, If i omit the filename on line 33 I get an error in vtkImageReader2.cxx:111 for not setting a filename.